src/profile/v100/address/non_fungible_local_id.rs

```diff
@@ -1,37 +1,50 @@
 use crate::prelude::*;
 
 use radix_engine_common::prelude::NonFungibleLocalId as ScryptoNonFungibleLocalId;
+use radix_engine_common::prelude::StringNonFungibleLocalId as ScryptoStringNonFungibleLocalId;
+use radix_engine_toolkit_json::models::common::SerializableNonFungibleLocalId as RetNonFungibleLocalId;
 
 #[derive(Clone, Debug, Hash, PartialEq, Eq, uniffi::Enum)]
 pub enum NonFungibleLocalId {
     Integer { value: u64 },
-    Str { value: String },
+    Str { value: NonFungibleLocalIdString },
```

Comment:
Using a type safe (our own) type allows for `From` instead of `TryFrom` of NonFungibleLocalId -> `Scrypto NonFungibleLocalId` (it has validates length and contents)